src/hooks/dependencies.ts

```diff
@@ -30,7 +30,13 @@ const registerInstallationHook = (
 
     if (installArg) {
```

Comment:
I think we can write the logic with this code:

```ts
if (typeof installArg === 'boolean') {
  installDeps = installArg
} else {
  installDeps = await confirm({
    message: 'Do you want to install project dependencies?',
    default: true,
  })
}
```

Reply:
You're right. I've made the change